2-collectors/scala-stream-collector/examples/config.hocon.sample

```diff
@@ -103,6 +119,7 @@ collector {
     }
 
     # Incoming events are stored in a buffer before being sent to Kinesis/Kafka.
+    # Note: Buffering is not supported by NSQ; will be ignored    
```

Comment:
> **output_buffer_size (nsqd v0.2.21+)** the size in bytes of the buffer nsqd will use when writing to this client.
This part is from [here](http://nsq.io/clients/tcp_protocol_spec.html)
As I understand, these buffer fields are used while nsqd is writing to the clients. I think there is no buffering option for producer. It is obvious from [here](https://github.com/brainlag/JavaNSQClient/blob/master/src/main/java/com/github/brainlag/nsq/NSQProducer.java#L28) too because NSQProducer class does not take any NSQConfig argument and creates default NSQConfig which buffer fields are null in the default NSQConfig.  I will add buffering to stream-enrich but I think this is not possible for stream-collector. What do you think ?